_targets.R

```diff
@@ -24,6 +24,35 @@ if (Sys.getenv("GCS_AUTH_FILE") != "") {
   repository <- targets::tar_option_get("repository")
 }
 
+# Model parameters
+dflt_pa <- c("log_pa_volume", "scale_pa_intensity")
+scale_pa <- c("scale_pa_volume", "scale_pa_intensity")
+dflt_sleep <- c(
+  "scale_sleep_duration", "scale_sleep_efficiency", "scale_sleep_onset",
+  "scale_sleep_regularity"
+)
+dflt_ranef <- "(1|studyid) + (1|participant_id)"
+fixedef <- "(1|measurement_day) + (1|participant_id)"
+dflt_con <- c("ses", "age", "sex", "bmi")
+
+# nolint start styler: off
+models_df <- dplyr::tribble(
+  ~model_name,          ~moderator,        ~mod_term,        ~mod_formal,        ~pa_vars, ~sleep_vars, ~ranef,     ~cont_vars,
+  "by_age",             "age",             "11, 18, 35, 65", "age",              dflt_pa,  dflt_sleep,  dflt_ranef, dflt_con,
+  "by_age_fixedef",     "age",             "11, 18, 35, 65", "age",              dflt_pa,  dflt_sleep,  fixedef,    c(dflt_con, "studyid"),
```

Comment:
Should we ditch the random intercept for measurement_day here as everyone starts on different days?